osu.Game/Online/Rooms/MultiplayerPlaylistItem.cs

```diff
@@ -39,6 +39,19 @@ public class MultiplayerPlaylistItem
         [Key(7)]
         public bool Expired { get; set; }
 
+        /// <summary>
+        /// The order in which this <see cref="MultiplayerPlaylistItem"/> will be played, starting from 0 and increasing for items which will be played later.
+        /// </summary>
+        [Key(8)]
+        public ushort PlaylistOrder { get; set; }
+
+        /// <summary>
+        /// The date when this <see cref="MultiplayerPlaylistItem"/> was last updated.
+        /// Not serialised to/from the client.
```

Comment:
Any reason it isn't? Seems like it'd be handy to have it (as discussed).

Reply:
Yep, will update it to be.